### PR TITLE
Fix build and cleanup

### PR DESCRIPTION
**Pull request by massdosage**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Eclipse files
```
Comment (massdosage) [inline .gitignore:1]: The project was missing a .gitignore entirely so I've added one.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: java
```
Comment (massdosage) [inline .travis.yml:1]: This should allow it to be built in Travis which gets us another CI environment to spot issues etc.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
-- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0)
+- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0).
+- Upgraded `hive-jdbc` to 2.3.4 (was 2.0.0).
+- Upgraded `hotels-oss-parent` to 4.0.1 (was 2.0.6).
+- Changed dependency from `com.microsoft.sqlserver.sqljdbc` (4.1) to `com.microsoft.sqlserver.sqljdbc4` (4.0).
```
Comment (massdosage) [inline CHANGELOG.md]: This was the main change I did to get it building locally, the other changes are nice to haves but put us in a better position for building further on this.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
-- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0)
+- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0).
+- Upgraded `hive-jdbc` to 2.3.4 (was 2.0.0).
```
Comment (massdosage) [inline CHANGELOG.md]: Hopefully this change is backwards compatible with how it's currently being used in production, will need to keep an eye on this when a new release is done and used for the first time. Ideally we'd test a SNAPSHOT build first.

File: pom.xml [hunk 5]
```diff
@@ -7,16 +7,17 @@
...
   </parent>
 
   <artifactId>quibble</artifactId>
   <version>1.0.1-SNAPSHOT</version>
   <name>Quibble</name>
 
   <scm>
-    <connection>scm:git:ssh://git@stash:7999/hdw/hww-expat.git</connection>
-    <developerConnection>scm:git:ssh://git@stash:7999/hdw/hww-expat.git</developerConnection>
+    <connection>scm:git:git@github.com:HotelsDotCom/quibble.git</connection>
```
Comment (massdosage) [inline pom.xml:18]: Don't know how this ever got pushed as it was TBH, anyway....

File: src/main/java/com/hotels/hde/quibble/Action.java [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2015-2018 Expedia Inc.
+ * Copyright (C) 2015-2019 Expedia, Inc.
```
Comment (massdosage) [inline src/main/java/com/hotels/hde/quibble/Action.java:2]: All the license headers get updated as a side effect of modifying the project, they're all the same - updating the year and adding a "," (which according to the legal department is more correct).

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,8 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Upgraded `hotels-oss-parent` to 4.0.1 (was 2.0.6).
+- Changed dependency from `com.microsoft.sqlserver:sqljdbc` (4.1) to `com.microsoft.sqlserver:mssql-jdbc` (6.4.0.jre7).
+- Changed dependency from `9.4.1208-jdbc42-atlassian-hosted` (4.1) to `org.postgresql:postgresql` (42.2.5.jre7).
 
+### Removed
+- DB2 and Teradata drivers (not available in Maven Central).
```
Comment (massdosage) [inline CHANGELOG.md:16]: These can't be in the pom unfortunately, the above changes will fix the build but I think the documentation and deployment procedure will need to be amended to explain how to get the drivers on the classpath when they're not open source and available in Maven Central. Going forward perhaps we shouldn't package any with Quibble and just tell people to bring their own, that can be in a different PR though.

File: pom.xml [hunk 8]
```diff
@@ -7,16 +7,17 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.0.6</version>
+    <version>4.0.1</version>
   </parent>
 
   <artifactId>quibble</artifactId>
-  <version>1.0.1-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```
Comment (massdosage) [inline pom.xml:14]: We're effectively making a non-backwards compatible change by removing some driver jars so I've bumped the major version.

File: CHANGELOG.md (hunk 3 above)
Comment (blakewhatley82) [reply CHANGELOG.md]: Yeah makes sense

File: pom.xml (hunk 5 above)
Comment (blakewhatley82) [reply pom.xml:18]: good find def need to be fixed

File: CHANGELOG.md (hunk 7 above)
Comment (blakewhatley82) [reply CHANGELOG.md:16]: Agree my thinking is to perhaps to package in our own version but the open source should not have them.

**Review by blakewhatley82 — COMMENTED** [review]
Changes look good. Spotted at bottom some issues about continuous-integration/travis-ci errors? Not sure if this is an issue or not

**Comment by massdosage** [comment]
> Changes look good. Spotted at bottom some issues about continuous-integration/travis-ci errors? Not sure if this is an issue or not

Yeah, the new parent changed default Java version to 8 but we're building it with Java 7 so that was the error. I've now set it to Java 7 for Quibble and this seems to have fixed it in Travis.

**Review by blakewhatley82 — APPROVED** [review]

**Pull request merged**